### ISSUE TITLE
Repo questions/issues

### ISSUE THREAD

**Issue by brettz9** [indexeddbshim/IndexedDBShim#264]
1. There is a link on the main page to "Apache-2.0" but it leads to the MIT license. Could you fix these either way, @axemclion (ideally indicating a dual license if Github allows so in the settings)?
2. Do you want to do anything with `gh-pages`? Should I keep it up to date?

**Comment by brettz9** [comment]
Also, Github now allows "topics" (visible at the top of the main page). It would be good for us to add the "indexeddb" topic to the repo for better discoverability.

**Comment by axemclion** [comment]
Agreed. Lets start with IndexeDB, WebSQl, Storage to start with ?

**Comment by brettz9** [comment]
Sounds good to me...

Regarding the license issue--to clarify, it was in regard to the "Apache-2.0" button that shows up on the repo but which links to our MIT license instead. I tested it on my fork to see if I could get rid of it, but only deleting the Apache license got it fixed. I reported the issue just now to Github and can report back.

But I would still like to know about gh-pages if there is anything you wanted to be done with it...

**Comment by brettz9** [comment]
@axemclion... Not sure whether you saw my comment at https://github.com/axemclion/IndexedDBShim/issues/262#issuecomment-296140986 ? We could really use an answer about a new release. Alternatively, if you don't have time to consider but trust us to go ahead, could you add me on for `npm publish` support so we can move forward?

**Comment by axemclion** [comment]
@brettz9 Sorry, my bad for falling back on this. I think you have been doing an amazing job on the repo and you should be the publisher. What should be the name of the npm package you would want to be the owner of ?

**Comment by brettz9** [comment]
Thanks, @axemclion . :-)

I was just thinking if you could add me as an owner of the current one you have been using so I can publish to the "indexeddbshim" package so that there is no break for those users who are allowing for major version semver breaks.

`npm owner add brettz9 indexeddbshim`

(The npm docs state there is no fine grain control of permissions, so I'd end up as a co-owner with the same privileges.)

If you mean you want to publish under a new package name though, of course, that would be cool too. Maybe "indexeddbpolyfill" would fit if you wanted to go that route. (My interpretation of "polyfill" is that you can use the same standard API--which fits for the default approach even better than "shim" I think.)

Whatever you decide, before we publish, I need to make a clear notice to any migrating users because it will break existing storage and use of existing storage. Code could have admittedly been added to allow migration of content, but it would have been ugly, probably add a performance cost as we'd need to check existing data each time, taken quite a bit more time to implement that I didn't care to spend, and I figured as the version was a breaking one with other breaking fixes anyways, that we could afford a clean break, assuming we at least convey the changes to users.

~Besides backward-breaking compatibility, there is also one area in which we might need to protect ourselves to avoid requiring more content-breaking changes in the future--key sorting type precedence is currently determined by searching the key type name by position within an array, and if IndexedDB ends up supporting new key types, e.g., as [boolean was proposed](https://github.com/w3c/IndexedDB/issues/76) or as I also proposed there, for adding `null` or `undefined` keys, then we'd need to add fractional keys or otherwise adjust the sorting precedence pattern, unless the new additions are all of lower precedence (which would be unlikely if boolean keys are added). I think I ought to fix this before our final release too so that stored data will be forward-compatible as far as sorting.~ *While this is a concern, I think we can put it off for now and refactor to allow for float values in between the integers if necessary.*

**Comment by axemclion** [comment]
Looks like I am not the owner of https://www.npmjs.com/package/indexeddbshim. @BigstickCarpet Can you please add @brettz9 as an owner on that package please ? 

If we don't hear from @BigstickCarpet I think you should publish a new package, as you have done most of the work, and you are probably now a more appropriate owner of the project than me.

**Comment by brettz9** [comment]
Ok, thanks, @axemclion . And unless there are any objections, I'll just update `gh-pages` with `master`. (And FWIW, it seems the Apache license icon is now fixed by Github to link to our Apache license)

**Comment by JamesMessinger** [comment]
@brettz9 and @axemclion - What are your NPM account names?  I'll add you both as owners of the `indexeddbshim` package

**Comment by brettz9** [comment]
Thanks, @BigstickCarpet . `brettz9` is mine.

**Comment by JamesMessinger** [comment]
lol.  I guess I should have assumed that you'd both use the same usernames for NPM as you do for GitHub.  :)

Anyway... you've both been added to the npm package

**Comment by brettz9** [comment]
Awesome, thanks! And now version 3.0.0-rc.1 is published to npm and released as a tag for Bower! (and `gh-pages` updated, though that is redirecting to http://nparashuram.com/IndexedDBShim/ ). Many thanks to the great foundation you guys set up! Hopefully the latest changes will be of use and work well, even with the cost in backward-compatibility...

Btw, one more question-- @axemclion -- the README and source maps are also currently pointing to http://nparashuram.com . Are you interested in having your site match our latest release?

In looking for possible alternatives in case you weren't interested, unfortunately, `unpkg.com` only includes the npm release (which doesn't include the `node_modules` files as are required in certain Mocha tests) and `rawgit.com` doesn't include `node_modules` either as they are not part of the repo either, so I'm not sure if there are any out of the box solutions.

**Comment by brettz9** [comment]
@axemclion : I went ahead and made the demo tests and source maps point to http://rawgit.com (I added a Grunt routine to copy the few test files needed out of `node_modules` into the repo itself). See https://cdn.rawgit.com/axemclion/IndexedDBShim/v3.0.0-rc.6/ . (Unfortunately for the latest W3C tests, those must currently be run locally (and with some set-up work).)

I think that having things on rawgit.com will be best as it ensures we can control everything in one place and keep it up to date.

I do see you have some code at your site (the jQuery plugin) that we don't have here, but as it is no doubt out of date, so I only borrowed its stylesheet (which was expected by `index.html`). The `index.html` page still points to your site, however.

I think I'm satisfied now to close the issue.
